content/en/case-studies/blackhole-image.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Case Study: The First Image of a Black Hole"
+sidebar: false
+---
+
+{{< figure src="/images/content_images/cs/blackhole.jpg" caption="**Black Hole M87**" alt="bhimg" attr="(**Image Credits:** Event Horizon Telescope Collaboration)" attrlink="https://www.jpl.nasa.gov/images/universe/20190410/blackhole20190410.jpg" >}}
```

Comment:
Minor but important for accessibility: could you make the `alt` text more descriptive? Something like "Black hole image"